src/pint/models/frequency_dependent.py

```diff
@@ -36,7 +36,7 @@ def __init__(self):
                 name="FD1",
                 units="second",
                 value=0.0,
-                description="Coefficient of delay as a polynomial function of log-frequency",
+                description="Coefficient of frequency-dependent delay",
```

Comment:
Again, why make the description *less* specific?

Reply:
Changed this.